lib/exec.ts

```diff
@@ -407,7 +407,7 @@ export async function sandbox(
 ): Promise<UnprocessedExecResult> {
     checkExecOptions(options);
     const type = execProps('sandboxType', 'firejail');
-    const dispatchEntry = sandboxDispatchTable[type];
+    const dispatchEntry = sandboxDispatchTable[type as 'none' | 'nsjail' | 'firejail' | 'cewrapper'];
```

Comment:
Seems this is the same issue as with the frontend settings, just less keys. Not really happy about it, but seems ok for now.